.changeset/hungry-coats-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Default to target: node16, so that top-level await is supported in user code
```

Comment:
```suggestion
Default to target: node14.8, so that top-level await is supported in user code
```

Reply:
some mismatch but I think we don't want to rewrite version in this part, instead set target in config to node16.